edit_book.php

```
@@ -0,0 +1,44 @@
+<?php
+include 'database.php';
+
+if (isset($_GET['id'])) {
+    $id = $_GET['id'];
+    $query = "SELECT * FROM books WHERE id = $id";
+    $result = mysqli_query($connection, $query);
```

The code does not handle errors returned by `mysqli_query`, which could lead to silent failures. Consider capturing the return value of the query and handling errors appropriately, possibly with exceptions.